nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/helpers/AbstractTestColdStorageHelper.java

```diff
@@ -19,20 +19,13 @@
 
 package org.nuxeo.coldstorage.helpers;
 
-import static javax.servlet.http.HttpServletResponse.SC_CONFLICT;
-import static javax.servlet.http.HttpServletResponse.SC_FORBIDDEN;
-import static javax.servlet.http.HttpServletResponse.SC_NOT_FOUND;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static javax.servlet.http.HttpServletResponse.*;
+import static org.junit.Assert.*;
```

Comment:
same thing about the https://doc.nuxeo.com/corg/intellij-idea-setup/#configure-nuxeo-code-style